src/test/ui/issues/issue-63226.rs

```diff
@@ -0,0 +1,10 @@
+// aux-build:issue-63226.rs
```

Comment:
Also, can we move this test to where the other tests for `const fn`s are?
A comment explaining the test would also be good. :)

Reply:
Moved into `ui/consts`